toolbox

```diff
@@ -269,7 +270,7 @@ copy_etc_profile_d_toolbox_to_container()
 
     echo "$base_toolbox_command: copying $toolbox_runtime_directory/toolbox.sh to container $container" >&3
 
-    if ! podman exec \
+    if ! $podman exec \
```

Comment:
We should quote these. eg., `"$podman".

Reply:
As we discussed, is it really necessary? In some places it isn't interpreted well (lines 1287, 1294,..).

Reply:
Yes, you are right.

I was worried about triggering [SC2086](https://github.com/koalaman/shellcheck/wiki/SC2086), but sometimes you just have to silence the warning as you can see (look for `SC2086`) from the code.